.github/workflows/ci.yml

```
@@ -12,7 +12,12 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: fetch main branch
+        run:  git fetch origin main:main
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)